idb/postgres/postgres_migrations.go

```diff
@@ -655,6 +656,7 @@ func processAccountTransactions(txnrows <-chan idb.TxnRow, addressStr string, ad
 
 		if accounting.AssetCreateTxn(stxn) {
 			result.asset[txn.AssetID] = updateCreate(result.asset[txn.AssetID], txn.Round)
+			result.assetHolding[txn.AssetID] = updateCreate(result.assetHolding[txn.AssetID], txn.Round)
```

Comment:
Accounting bug 1, we weren't marking deleted = false for the creator's asset holding.